pkgs/applications/networking/mailreaders/thunderbird-bin/default.nix

```diff
@@ -169,7 +169,7 @@ stdenv.mkDerivation {
 
   passthru.updateScript = import ./../../browsers/firefox-bin/update.nix {
     inherit writeScript xidel coreutils gnused gnugrep curl gnupg runtimeShell;
-    name = "thunderbird-bin-${version}";
+    pname = "thunderbird-bin";
```

Comment:
I don't see an `inherit version;`?

Reply:
It is used as a name of a shell script.  Version is not required.

`pkgs/applications/networking/browsers/firefox-bin/update.nix`:
```
in writeScript "update-${pname}" ''
```

Reply:
Right, thanks